packages/test-base/src/androidAndroidTest/kotlin/io/realm/kotlin/test/shared/RealmAnyTests.kt

```diff
@@ -0,0 +1,119 @@
+/*
```

Comment:
Wouldn't this file be part of another PR?

Reply:
I see, if they are part of other PR I think it is better to move them. But I see that there is some value to have some specific tests for the transport. Couldn't we add some tests that validate the different scopes?